packages/puppeteer-core/src/api/ElementHandle.ts

```diff
@@ -897,7 +963,155 @@ export class ElementHandle<
+      const styleBox = {
+        padding: {
+          left: parseInt(style.paddingLeft, 10),
+          top: parseInt(style.paddingTop, 10),
+          right: parseInt(style.paddingRight, 10),
+          bottom: parseInt(style.paddingBottom, 10),
+        },
+        margin: {
+          left: parseInt(style.marginLeft, 10),
+          top: parseInt(style.marginTop, 10),
+          right: parseInt(style.marginRight, 10),
+          bottom: parseInt(style.marginBottom, 10),
+        },
+        border: {
+          left: parseInt(style.borderLeft, 10),
+          top: parseInt(style.borderTop, 10),
+          right: parseInt(style.borderRight, 10),
+          bottom: parseInt(style.borderBottom, 10),
+        },
+      };
+      const border: Quad = [
+        {x: rect.left, y: rect.top},
+        {x: rect.left + rect.width, y: rect.top},
+        {x: rect.left + rect.width, y: rect.top + rect.bottom},
+        {x: rect.left, y: rect.top + rect.bottom},
+      ];
+      const padding: Quad = [
+        {
+          x: border[0].x + styleBox.border.left,
+          y: border[0].y + styleBox.border.top,
+        },
+        {
+          x: border[1].x - styleBox.border.right,
+          y: border[1].y + styleBox.border.top,
+        },
+        {
+          x: border[2].x - styleBox.border.right,
+          y: border[2].y - styleBox.border.bottom,
+        },
+        {
+          x: border[3].x + styleBox.border.left,
+          y: border[3].y - styleBox.border.bottom,
+        },
+      ];
+      const content: Quad = [
+        {
+          x: padding[0].x + styleBox.padding.left,
+          y: padding[0].y + styleBox.padding.top,
+        },
+        {
+          x: padding[1].x - styleBox.padding.right,
+          y: padding[1].y + styleBox.padding.top,
+        },
+        {
+          x: padding[2].x - styleBox.padding.right,
+          y: padding[2].y - styleBox.padding.bottom,
+        },
+        {
+          x: padding[3].x + styleBox.padding.left,
+          y: padding[3].y - styleBox.padding.bottom,
+        },
+      ];
+      const margin: Quad = [
+        {
+          x: border[0].x - styleBox.margin.left,
+          y: border[0].y - styleBox.margin.top,
+        },
+        {
+          x: border[1].x + styleBox.margin.right,
+          y: border[1].y - styleBox.margin.top,
+        },
+        {
+          x: border[2].x + styleBox.margin.right,
+          y: border[2].y + styleBox.margin.bottom,
+        },
+        {
+          x: border[3].x - styleBox.margin.left,
+          y: border[3].y + styleBox.margin.bottom,
+        },
+      ];
+      return {
+        content,
+        padding,
+        border,
+        margin,
+        width: rect.width,
+        height: rect.height,
+      };
+    }, adoptedThis);
+    void adoptedThis.dispose().catch(debugError);
+    if (!model) {
+      return null;
+    }
+    let frame: Frame | null | undefined = this.frame;
+    let element: HandleFor<HTMLIFrameElement> | null | undefined;
```

Comment:
Remove undefined init as null or use do {} while. Same for above frame should be Frame or null, never undefined.

Reply:
It can be `undefined` because of nullish collescing (`?.`).